setup.py

```diff
@@ -20,12 +20,14 @@
         'Stanislav Kudriashev',
         'Raoul Snyman',
         'Corey Goldberg',
+        'Harald Nordgren',
     ]),
     url='https://github.com/mahmoudimus/nose-timer',
     packages=['nosetimer', ],
     install_requires=[
         'nose',
         'termcolor',
+        'ordereddict',
```

Comment:
Proposing it be dropped as Python 2.6 has already been dropped. ( https://github.com/mahmoudimus/nose-timer/pull/84 )